dali/pipeline/operators/python_function/python_function.cc

```diff
@@ -0,0 +1,88 @@
+#include "python_function.h"
+
+namespace dali {
+
+DALI_SCHEMA(PythonFunction)
+        .DocStr(R"code(Executes python function that consumes and produces single numpy array.)code")
+        .NumInput(1)
+        .NumOutput(1)
+        .AllowMultipleInputSets()
+        .EnforceInputLayout(DALI_NHWC)
+        .AddArg("function_id",
+                R"code(Id of the python function.)code",
+                DALI_INT64);
+
+struct PyBindInitializer {
+  PyBindInitializer() {
+    auto thread_state = PyGILState_Ensure();
+    pybind11::get_shared_data(""); // setup the pybind's internals pointer
+    PyGILState_Release(thread_state);
+  }
+};
+
+// Some pybind's internals are not properly initialized when used in dynamically linked library,
+// so this workaround initializes them manually
+static PyBindInitializer pybind_initializer{}; // NOLINT
+
+py::array tensorToNumpyArray(const Tensor<CPUBackend> &tensor) {
```

Comment:
Use camel case with first letter Uppercase in functions

Reply:
+1   We use CamelCase and sometimes snake_case (especially in utilities that follow STL style)